proco/connection_statistics/api.py

```diff
@@ -42,6 +44,18 @@ def get(self, request, *args, **kwargs):
             'countries_connected_to_realtime': aggregate_statuses['countries_connected_to_realtime'],
             'countries_with_static_data': aggregate_statuses['countries_with_static_data'],
         }
+
+        cache_data = cache.get(self.CACHE_KEY_LAST_DATA_UPDATE)
```

Comment:
maybe it would be better just to decorate get method with cache instead of trying to save partially values?